sympy/series/limits.py

```diff
@@ -124,6 +124,11 @@ def doit(self, **hints):
         from sympy.series.limitseq import limit_seq
 
         e, z, z0, dir = self.args
+        
+        if type(e) is exp:
+            for b in e.atoms(Symbol):
```

Comment:
After modifying this and  it is returning correct answer for few examples and also passes all checks for `test_limits.py` in my system.May be because there is no test case like `exp(x**a)*a`. I want to see the travis result.
I think I should use `args` first the check for `exp` or `Pow` type using `atoms(Symbols)` we will get base and it's Power symbols then we can check for `is_zero` .
As I said in [previous comment](https://github.com/sympy/sympy/pull/10385#issuecomment-171998317).
I hope this way is correct.